ui/app/helpers/mountable-secret-engines.js

```diff
@@ -9,7 +9,7 @@ const ENTERPRISE_SECRET_ENGINES = [
   {
     displayName: 'KMIP',
     type: 'kmip',
-    engineRoute: 'kmip.scopes',
+    engineRoute: 'kmip.scopes.index',
```

Comment:
I had added this, along with a redirect route from the scopes engine and then had subsequently removed it - that commit must have failed.